From 10b3508bd3aad4a5dc3e37f3b7bcd143ddad9c43 Mon Sep 17 00:00:00 2001
From: Vincent Bernat <vincent@bernat.im>
Date: Sat, 28 May 2016 15:06:44 +0200
Subject: Use PyYAML instead of poyo

There is no problem with PyYAML in Debian and therefore, we don't need
to switch to poyo (a pure YAMLish parser).
---
 cookiecutter/config.py | 6 +++---
 setup.py               | 2 +-
 2 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/cookiecutter/config.py b/cookiecutter/config.py
index 060e6ca98a82..cd3bd23481e9 100644
--- a/cookiecutter/config.py
+++ b/cookiecutter/config.py
@@ -13,7 +13,7 @@ import logging
 import os
 import io
 
-import poyo
+import yaml
 
 from .exceptions import ConfigDoesNotExistException
 from .exceptions import InvalidConfiguration
@@ -54,8 +54,8 @@ def get_config(config_path):
     logger.debug('config_path is {0}'.format(config_path))
     with io.open(config_path, encoding='utf-8') as file_handle:
         try:
-            yaml_dict = poyo.parse_string(file_handle.read())
-        except poyo.exceptions.PoyoException as e:
+            yaml_dict = yaml.safe_load(file_handle.read())
+        except yaml.error.YAMLError as e:
             raise InvalidConfiguration(
                 'Unable to parse YAML file {}. Error: {}'
                 ''.format(config_path, e)
diff --git a/setup.py b/setup.py
index 5a45b5c3df0f..2805cd0a858e 100644
--- a/setup.py
+++ b/setup.py
@@ -28,7 +28,7 @@ requirements = [
     'jinja2>=2.7',
     'click>=5.0',
     'whichcraft>=0.4.0',
-    'poyo>=0.1.0',
+    'PyYAML',
     'jinja2-time>=0.1.0'
 ]
 
